governance/votes/Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,202 @@
+
+- A new Output Conduit at [0x1E86CB085f249772f7e7443631a87c6BDba2aCEb](https://etherscan.io/address/0x1E86CB085f249772f7e7443631a87c6BDba2aCEb) will be authorized, allowing the `operator` to select which PSM BlockTower Andromeda transactions are routed through. Operationally, this means that USDC, GUSD, or USDP may be used for drawdowns.
+- The relevant [Chainlog](https://chainlog.makerdao.com/) entry `RWA015_A_OUTPUT_CONDUIT` will be updated to reflect the change to the Output Conduit.
+- The USDP, GUSD, and USDC PSMs will be whitelisted for use by calling `clap`.
+- The current Output Conduit will be deauthorized.
+- The old `RWA015_A_OUTPUT_CONDUIT_LEGACY` will be removed from the chainlog as it will no longer be used.
+
+### Smart Burn Engine Deployment
+
+As per this successful govenrance [poll](https://vote.makerdao.com/polling/QmQmxEZp) and the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#9-surplus-buffer-and-smart-burn-engine) a new Smart Burn Engine will be deployed along with the following parameters, if this executive proposal passes.
+
+- Deploy the new Smart Burn Engine as `MCD_FLAP`.
+- The [Maximum System Surplus (`vow.hump`)](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-auction-limit) will be reduced by 200 million DAI from 250 million DAI to **50 million DAI**.
+- The [Surplus Auction Lot Size (`vow.bump`)](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-lot-size) will be reduced by 25,000 DAI from 30,000 DAI to **5,000 DAI**.
+- The `hop` will be set to **1,577 seconds**. The `hop` parameter defines the minimum amount of time that must elapse between `kick`s, therefore setting the maximum frequency of market actions.
+- The `want` will be set to **0.98**. The `want` parameter defines the relationship between the purchase price of MKR and the price provided by the MKR/USD oracle, i.e. the slippage. A want of 0.98 means that the purchase price of MKR can be up to 2% worse than the oracle price. However, if the price of MKR on Uniswap is better than the Oracle price, the Smart Burn Engine will not be constrained.
+- The `pip` will be set to `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df`. The `pip` is the oracle referenced by the Smart Burn Engine.
+- The `receiver` will be set to the Pause Proxy address - `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df `. The `receiver` is the address that LP tokens will be deposited to after each market action.
```

Comment:
Receiver address is incorrect - should be 0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB

Etherscan links for both pip and receiver address?